### PR TITLE
Directory structure for the import automation project

### PR DESCRIPTION
**Pull request by intrepiditee**
import-automation
    - cloudbuild
    - executor
    - progress-dashboard-rest

### CODE REVIEW

File: import-automation/cloudbuild/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Cloud Build Configuration and Code
+
+This directory contains configuration and code for Cloud Build that
+enables it to create asynchronous tasks using Cloud Tasks on pull
+request and push to master.
```
Comment (jeffreyoldham) [inline import-automation/cloudbuild/README.md:5]:
```suggestion
requests and pushes to master.
```

File: import-automation/executor/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Import Executor
+
+This directory contains code of an App Engine app that exposes an endpoint
```
Comment (jeffreyoldham) [inline import-automation/executor/README.md:3]:
```suggestion
This directory contains code for an App Engine app that exposes an endpoint
```

File: import-automation/cloudbuild/README.md (hunk 1 above)
Comment (intrepiditee) [reply import-automation/cloudbuild/README.md:5]: Fixed

File: import-automation/executor/README.md (hunk 2 above)
Comment (intrepiditee) [reply import-automation/executor/README.md:3]: Fixed

**Pull request merged**